packages/jtex/src/imports.ts

```diff
@@ -11,13 +11,20 @@ function label(title: string, commands: string[]) {
   return `${titleBlock}${commands.join('\n')}\n`;
 }
 
-export function createImportCommands(commands: Set<string>, existingPackages?: string[]) {
+export function createImportCommands(commands: Set<string>, existingPackages?: string[]): string[] {
   const sorted = [...commands].sort();
   const existingSet = new Set(existingPackages);
   const filtered = existingPackages ? sorted.filter((p) => !existingSet.has(p)) : sorted;
   return filtered.map((c) => `\\usepackage{${c}}`);
 }
 
+export function createGlossaryDirectives(directives: GlossaryDirective[]): string[] {
+  const usepackage = '\\usepackage{glossaries}';
+  const makeglossaries = '\\makeglossaries';
```

Comment:
Does everything here need to be in the preamble with the imports? Or could we move `\makeglossaries`,`\newglossaryentry`, etc to `myst-to-tex` and just write it before everything else?

(Looking briefly at the docs, it appears `\makeglossaries` only needs to come before the first glossary definition? But I know LaTeX is quite picky...)